### ISSUE TITLE
Please add an example

### ISSUE THREAD

**Issue by j-funk** [benjamine/page-object-pattern#1]
This looks great, a small example app which uses this would make picking this up a lot faster.

**Comment by benjamine** [comment]
absolutely, sorry, this is extracted from a real private project, but the proper work for open sourcing (like documentation and unit tests) was never done, but it definitely should.

I plan to add decent unit test coverage and  an example project using this with cucumber and leadfoot as browser automation framework, stay tuned!

**Comment by karlbecker** [comment]
Still would love to see this!
